bigquery/scheduled_queries/calculate-daily-vacancy-tag-time-metrics.sql

```diff
@@ -221,7 +221,39 @@ IF
   (date>='2020-08-01',
     SAFE_DIVIDE(live_vacancies_with_this_tag,
       live_benchmark_vacancies_with_this_tag),
-    NULL) AS live_vacancies_with_this_tag_as_proportion_of_benchmark
+    NULL) AS live_vacancies_with_this_tag_as_proportion_of_benchmark,
+  SAFE_SUBTRACT(SAFE_DIVIDE(live_vacancies_with_this_tag,
+      total_live_vacancies),
+    SAFE_DIVIDE(live_vacancies_with_this_tag_last_year,
+      total_live_vacancies_last_year)) AS change_in_proportion_of_live_vacancies_with_this_tag_since_last_year,
+IF
+  (date>='2021-08-01',
```

Comment:
I'm not sure why the date comparison has been updated to 2021. What date is this? (Can't immediately seem to figure it out myself.)

Reply:
It's because comparisons to a year ago won't be meaningful until we have at least a year's worth of data, which will be on this date in 2021. I'll add a comment.